### PR TITLE
add module for updating jank index

### PR DESCRIPTION
**Pull request by amonks**
### how did i test this?

lol

### CODE REVIEW

**Comment by shaneschulte** [comment]
this is imperative to my workflow, please merge

**Comment by shaneschulte** [comment]
the CI build didn't fail so I think it's safe to merge

File: src/jank/jank.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const got = require("got");
+
+const jankRe = /jank:[0-9]+/;
```
Comment (shaneschulte) [inline src/jank/jank.js]: missing capture group :(

Comment (shaneschulte) [reply src/jank/jank.js]: fixed :)

**Comment by amonks** [comment]
very disappointed this has been sitting open this long

when can we expect a new release?

**Pull request merged**